docs/extend/modular-resources/_index.md

```diff
@@ -10,679 +10,26 @@ aliases:
     - "/program/extend/modular-resources/"
 ---
 
-The Viam module system allows you to integrate custom {{< glossary_tooltip term_id="resource" text="resources" >}}, ([components](/components/), and [services](/services/)) into any robot running on Viam.
+The Viam module system allows you to integrate custom {{< glossary_tooltip term_id="resource" text="resources" >}} into any robot running on Viam.
+The {{< glossary_tooltip term_id="rdk" text="RDK">}} [manages](/extend/modular-resources/key-concepts/) these modular resources like the resource types and models that are built-in to the kit.
 
-With modular resources, you can:
+Modular resources can be:
 
-- Create new models of built-in component or service types
-- Create brand new resource types
+1. New models of built-in [components](/components/) or [services](/services/) that implement the built-in resource type's API through Viam's [client SDKs](/program/apis/).
+2. Brand new types of resources that define their own API in [protocol buffers](https://developers.google.com/protocol-buffers).
 
-`viam-server` [manages](#modular-resource-management) modular resources configured on your robot like resources that are already built into the [Robot Development Kit (RDK)](/internals/rdk/).
+## Get Started
 
-Two key concepts exist across all Viam resources (both built-in and modular) to facilitate this: [*APIs*](#apis) and [*models*](#models).
+To get started adding custom resources to your robot, learn the [key concepts](/extend/modular-resources/key-concepts/) behind Viam's resource APIs that make the module system possible.
 
-## Key concepts
+Then, follow one of these [example tutorials](/extend/modular-resources/examples/) to add previously built modules and modular resources to your robot.
```

Comment:
But some of these examples show creating new models? Perhaps it would make more sense to point them to the extend/modular-resources/configure/ page instead? And if necessary, that page could have links to specifically relevant examples.

Reply:
What do you mean? It's the rplidar and odrive, which I think are good simple pages to show the configuration process for pre-built modules. They do offer new models but that's in the module so I don't catch your drift

Reply:
Right now, the page looks like this: https://docs.viam.com/extend/modular-resources/examples/ It is true that those pre-built modules are contained in the examples folder, but we also have links to tutorials there that create new modules.

I think you should do this:
```suggestion
To add a previously built module to your robot, [configure a modular resource](/extend/modular-resources/configure/].
```

Then, if you want, links to relevant examples can be added to the bottom of the configure page.

Also, somewhat less importantly, this suggestion changes the order of words such that people know what they're doing before clicking the link (versus telling them to click the link before we actually know whether they want to use a prebuilt module or to make a new one).